draft-regext-rdap-extensions.md

```diff
@@ -270,6 +391,30 @@ to technologies used by RDAP such as extended vCard/jCard properties.
 Such extensions exist to "mark" these usages and are often called "marker"
 extensions.
 
+For example, an extension may be used to signal desired processing of
+a `rel` attribute in a "links" array, where the `rel` value is registered in
+the Link Relations Registry (<https://www.iana.org/assignments/link-relations/link-relations.xhtml>).
+
+    {
+      "rdapConformance" : [
+        "rdap_level_0",
+        "lunarNIC"  
+      ],
+      "objectClassName" : "domain",
+      "ldhName": "example.com",
+      "links": [
+        {
+          "value": "https://example.com/domain/example.com",
+          "href" : "https://example.com/sideways_href",
+          "rel": "sideways",
+          "type": "application/rdap+json"
+        }
+      ]
+    }
+
+When defining the usage link relations, extensions SHOULD specify the
+media types expected to be used with those link relations.
```

Comment:
SHOULD seems strong here, given that there could be any number of link relations usable with arbitrary media types.